website_sale_hide_price/data/product_snippet_template_data.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
```

Comment:
@pilarvargas-tecnativa remove `noupdate` as per https://github.com/OCA/e-commerce/pull/1087 and move the file to another path.